src/Components/Nav/Nav.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
```

Comment:
Thanks for adding the Nav component. Let's try to get @BlueNealis's flamingo up there.